en/option/component/title.md

```diff
@@ -15,7 +15,7 @@ In ECharts 2.x, a single instance of ECharts could contains one title component
 
 ## show(boolean) = true
 
-It specifies whether to show the title component.
+Change this to `false` to prevent the title from showing

```

Comment:
I guess it's better to use the same expression as others, probably `Set this ...`?

Reply:
Oh yes! let's keep it consistent